README.md

```diff
@@ -1,5 +1,7 @@
 # bedrock-consumer
 
+[![Build Status](http://ci.digitalbazaar.com/buildStatus/icon?job=bedrock-consumer&build=5)](http://ci.digitalbazaar.com/job/bedrock-consumer/5/)
```

Comment:
good catch. Evidently you have to watch out where you copy the automatically generated status code from.  Fixed now.